### DISCUSSION TITLE
Most downloads are interrupted

### DISCUSSION THREAD

**Discussion by Gianfnc** [anidl/multi-downloader-nx#115] [Q&A]
Hey! Since I started using the app I've been having problems with interrupted downloads. Usually they always start downloading normally and randomly stop, the percentage doesn't advance, and I have to close the cmd, open it again, log in, and put it to download again.
![Sem título (1)](https://user-images.githubusercontent.com/80010274/133948670-fb3f1139-ed71-4c32-b237-40678c21b5d0.png)

I've done tests with the firewall and Windows Defender turned on and off, but it doesn't seem to make a difference. The account I use to log in has Premium Plus, and I don't have problems with connection either.
![Sem título (2)](https://user-images.githubusercontent.com/80010274/133948739-cba163e7-c4c1-4ada-90a1-9a4e311d9da8.png)

Have you had any similar problems? Is there any command to resume downloads that might eventually be interrupted? It usually takes a long time to download a full episode because I have to keep downloading again manually.

**Comment by OryWareKyrie** [comment]
which version are you using? try using the latest one. the @izu-co corrected this, apparently. I haven't tested it yet.

**Reply by Gianfnc** [reply]
I'm using 4.10.2, I saw now that you mentioned 4.12.2, I'll do a test and post it here.

**Reply by Gianfnc** [reply]
The problem is gone with version 4.12.2! Apparently now he can wait for a response and resume downloading, when in the previous version he couldn't reconnect.
![Sem título](https://user-images.githubusercontent.com/80010274/133964004-3bf2e75c-0ea9-488c-a294-177602736d40.png)
